Arc/Arc.js

```diff
@@ -0,0 +1,142 @@
+import kind from '@enact/core/kind';
```

Comment:
We need the module documentation like Button with @example at the first line.
```
/**
 * Sandstone styled button components and behaviors.
 *
 * @example
 * <Button>Hello Enact!</Button>
 *
 * @module sandstone/Button
 * @exports Button
 * @exports ButtonBase
 * @exports ButtonDecorator
 */
```

Reply:
I pushed  the fixes